File: 3304. Find the K-th Character in String Game I/kthCharacter.js
/**
 * @param {number} k
 * @return {character}
 */
var kthCharacter = function (k) {
    let word = "a"

    while (word.length < k) {
        let curr = ''
        for (let chr of word) {
            curr += String.fromCharCode((chr.charCodeAt(0) + 1) % 122)
        }
        word += curr
    }
    return word[k - 1]
};

File: 3304. Find the K-th Character in String Game I/kthCharacter.cpp
#include <iostream>

using namespace std;

class Solution {
    public:
        char kthCharacter(int k) {
            string word = "a";
    
            while(word.size() < k){
                string curr = "";
                for(auto chr : word){
                    curr += chr+1;
                }
                word+=curr;
            }
    
            return word[k-1];
        }
    };